apps/dashboard/test/models/user_configuration_test.rb

```diff
@@ -79,6 +79,7 @@ def teardown
       interactive_apps_menu: [],
       custom_pages: {},
       support_ticket: {},
+      apps_datatable: {},
```

Comment:
Looks like `:page_length` is going to be nil here.  I assume it's a hash so that we can further add things to it. In any case, seems like it should initialize the page_length key to maybe 10 or so.

Reply:
I totally missed that. I had it in a previous implementation, but after a refactoring I missed.
Added the default value for `page_length`